tests/user_commands/management/commands/dance.py

```diff
@@ -5,6 +5,7 @@ class Command(BaseCommand):
     help = "Dance around like a madman."
     args = ''
     requires_system_checks = True
+    private_options = ('opt_3', )
```

Comment:
This shouldn't be a private option. The option validation needs to be adjusted to allow this as described in 2cc8ffe.